crates/torii/core/src/model.rs

```diff
@@ -180,6 +183,10 @@ pub fn build_sql_query(
              [{model_table}].{entity_relation_column}",
         ));
 
+        if internal_updated_at > 0 {
+            internal_updated_at_clause.push(format!("[{model_table}].internal_updated_at >= ?"));
```

Comment:
Here there's a potential speedup if we can trust the `updated_at` that is inside the `entities` table. Which could avoid having this `WHERE` clause added for each models.